CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Handle `<base href="...">` values correctly when inlining imports.
 - Apply `<base target="...">` to links and forms in the same document as
   appropriate.
+- Fixed a bug where element attributes inside `<template>` tags were not
```

Comment:
You're right.  I should have set the base on the PR differently.  I think I was hoping to get the other one in master first and then the extra stuff would disappear before this was looked at.  

Now I've rebased the one I cut this from so I've rebased this one too.